src/main/pg/gps_rescue.h

```diff
@@ -27,25 +27,25 @@
 typedef struct gpsRescue_s {
 
     uint16_t maxRescueAngle; // degrees
-    uint16_t initialAltitudeM; // meters
+    uint16_t returnAltitudeM; // meters
     uint16_t descentDistanceM; // meters
-    uint16_t rescueGroundspeed; // centimeters per second
+    uint16_t groundSpeed; // centimeters per second
```

Comment:
You can add a hint for the unit to `groundSpeed`
```suggestion
    uint16_t groundSpeedCmS; // centimeters per second
```

Reply:
Thanks. Done.